cluster/gce/gci/configure-helper.sh

```diff
@@ -2896,7 +2896,7 @@ function main() {
   if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
     KUBE_BOOTSTRAP_TOKEN="$(secure_random 32)"
   fi
-  if [[ "${ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
+  if [[ "${EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
```

Comment:
This (the token) is about running the actual konnectivity service not about egress via the service. It determines things like if the kubeconfig for the konnectivity server is setup and we need it setup.

Reply:
Hmm, we should probably use a third variable (eg: `SETUP_KONNECTIVITY_SERVICE`) to handle this setup phase since it should not be bundled with the apiserver egress flag. 

An alternative is using an enum, but I think it would get a bit messy given that we would have very long enums to describe the combinations of variables currently.